test_scripts/Defects/5_0/2380_OnVehicleData_PoliciesManager_must_allow_all_requested_params_in_case_parameters_field_is_omited.lua

```diff
@@ -0,0 +1,258 @@
+    auxECallNotificationStatus = "NORMAL",
+    eCallConfirmationStatus = "NORMAL"
+  },
+  airbagStatus = {
+    driverAirbagDeployed = "NOT_SUPPORTED",
+    driverSideAirbagDeployed = "NOT_SUPPORTED",
+    driverCurtainAirbagDeployed = "NOT_SUPPORTED",
+    passengerAirbagDeployed = "NOT_SUPPORTED",
+    passengerCurtainAirbagDeployed = "NOT_SUPPORTED",
+    driverKneeAirbagDeployed = "NOT_SUPPORTED",
+    passengerSideAirbagDeployed = "NOT_SUPPORTED",
+    passengerKneeAirbagDeployed = "NOT_SUPPORTED"
+  },
+  emergencyEvent = {
+    emergencyEventType = "NO_EVENT",
+    fuelCutoffStatus = "NORMAL_OPERATION",
+    rolloverEvent = "NO_EVENT",
+    maximumChangeVelocity = 0,
+    multipleEvents = "NO_EVENT"
+  },
+  clusterModeStatus = {
+    powerModeActive = true,
+    powerModeQualificationStatus = "POWER_MODE_UNDEFINED",
+    carModeStatus = "TRANSPORT",
+    powerModeStatus = "KEY_OUT"
+  },
+  myKey = {
+    e911Override = "NO_DATA_EXISTS"
+  }
+}
+
+local VDValues = {
+  gps = "VEHICLEDATA_GPS",
+  speed = "VEHICLEDATA_SPEED",
+  rpm = "VEHICLEDATA_RPM",
+  fuelLevel = "VEHICLEDATA_FUELLEVEL",
+  fuelLevel_State = "VEHICLEDATA_FUELLEVEL_STATE",
+  fuelRange = "VEHICLEDATA_FUELRANGE",
+  instantFuelConsumption = "VEHICLEDATA_FUELCONSUMPTION",
+  externalTemperature = "VEHICLEDATA_EXTERNTEMP",
+  turnSignal = "VEHICLEDATA_TURNSIGNAL",
+  prndl = "VEHICLEDATA_PRNDL",
+  tirePressure = "VEHICLEDATA_TIREPRESSURE",
+  odometer = "VEHICLEDATA_ODOMETER",
+  beltStatus = "VEHICLEDATA_BELTSTATUS",
+  electronicParkBrakeStatus = "VEHICLEDATA_ELECTRONICPARKBRAKESTATUS",
+  bodyInformation = "VEHICLEDATA_BODYINFO",
+  deviceStatus = "VEHICLEDATA_DEVICESTATUS",
+  driverBraking = "VEHICLEDATA_BRAKING",
+  wiperStatus = "VEHICLEDATA_WIPERSTATUS",
+  headLampStatus = "VEHICLEDATA_HEADLAMPSTATUS",
+  engineTorque = "VEHICLEDATA_ENGINETORQUE",
+  engineOilLife = "VEHICLEDATA_ENGINEOILLIFE",
+  accPedalPosition = "VEHICLEDATA_ACCPEDAL",
+  steeringWheelAngle = "VEHICLEDATA_STEERINGWHEEL",
+  eCallInfo = "VEHICLEDATA_ECALLINFO",
+  airbagStatus = "VEHICLEDATA_AIRBAGSTATUS",
+  emergencyEvent = "VEHICLEDATA_EMERGENCYEVENT",
+  clusterModeStatus = "VEHICLEDATA_CLUSTERMODESTATUS",
+  myKey = "VEHICLEDATA_MYKEY"
+}
+
+-- [[ Local Functions ]]
+local function pTUpdateFunc(tbl)
+  local VDgroup = {
+    rpcs = {
+      SubscribeVehicleData = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },    
+      },
+      OnVehicleData = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
+      }
+    }
+  }
+tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups = {"Base-4", "NewTestCaseGroup"}
+end
+
+local function setVDRequest()
+  local tmp = {}
+    for k, _ in pairs(VDValues) do
+      tmp[k] = true
+    end
+  return tmp
+end
+
+local function setVDResponse()
+  local temp = { }
+  local vehicleDataResultCodeValue = "SUCCESS"
```

Comment:
@TMyza I propose to remove variable `vehicleDataResultCodeValue` because it uses only once in expression `resultCode = vehicleDataResultCodeValue` and to use its value ("SUCCESS") in mentioned expression instead.